docs/erasure/storage-class/README.md

```diff
@@ -9,7 +9,30 @@ set before starting Minio server. After the data and parity disks for each stora
 you can set the storage class of an object via request metadata field `x-amz-storage-class`. Minio server then honors the storage class by
 saving the object in specific number of data and parity disks.
 
-### Values for standard storage class (STANDARD)
+## Storage usage
+
+The selection of varying data and parity drives has a direct impact on the drive space usage. With storage class, you can optimise for high
+redundancy or better drive space utilisation.
+
+To get an idea of how various combinations of data and parity drives affect the storage usage, let’s take an example of a 100 MiB file stored
+on 16 drive Minio deployment. If you use eight data and eight parity drives, the file space usage will be approximately twice, i.e. 100 MiB
+file will take 200 MiB space. But, if you use ten data and six parity drives, same 100 MiB file takes around 160 MiB. If you use 14 data and
+two parity drives, 100 MiB file takes only approximately 116 MiB.
+
+| Total Drives | Data Drives | Parity Drives | Storage Usage Ratio |
+|--------------|-------------|---------------|---------------------|
+|           16 |           8 |             8 |                2.04 |
+|           16 |           9 |             7 |                1.82 |
+|           16 |          10 |             6 |                1.60 |
+|           16 |          11 |             5 |                1.45 |
+|           16 |          12 |             4 |                1.38 |
+|           16 |          13 |             3 |                1.24 |
+|           16 |          14 |             2 |                1.16 |
+
+This table summarizes the data/parity drives and corresponding storage space on a 16 drive Minio deployment. The field storage usage ratio is simply 
```

Comment:
providing a mathematical expression to define storage usage ratio would be easier to understand.

Reply:
sure, will add